task/buildah/0.2/buildah.yaml

```diff
@@ -385,7 +385,7 @@ spec:
         echo "Adding activation key to the build"
 
 
-        if ! grep -E "^[^#]*subscription-manager.[^#]*register" "$dockerfile_path"; then
+        if [ "${HERMETIC}" != "true" ] && ! grep -E "^[^#]*subscription-manager.[^#]*register" "$dockerfile_path"; then
```

Comment:
nitpick: it's not needed on this line, we only get here if hermetic is disabled (and activation key exists)

Reply:
fixed, thanks